tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -120,27 +120,50 @@ Feature: add users to group
       | Alice    | var/../etc       |
 
 
-  Scenario: normal user tries to add himself to a group
-    Given group "groupA" has been created
+  Scenario Outline: normal user tries to add himself to a group
```

Comment:
```suggestion
  Scenario Outline: user with different role tries to add ownself to a group
```
since user is of different role can we have a description like this? if its okay than can be changed to other scenarios as well.

Reply:
I am not confident about `user with a different role`.